fuel-vm/src/tests/validation.rs

```diff
@@ -35,3 +52,118 @@ fn transaction_can_be_executed_after_maturity() {
         .execute_tx(tx);
     assert!(result.is_ok());
 }
+
+/// Malleable fields should not affect validity of the block
+#[test]
+fn malleable_fields_do_not_affect_validity() {
```

Comment:
Awesome test!=) Implement manual zeroing fields and passing chain id to calculate the hash of the transaction from its bytes is huuuuuuge